README.md

```diff
@@ -121,106 +121,10 @@ If you choose Use Advanced Image Configurations, you can click on Advanced butto
-.build()
-
-Jenkins.getInstance().clouds.add(myCloud)
-```
-```groovy
-//Configure cloud with mutli-template of advanced images
-import com.microsoft.azure.vmagent.builders.*
-
-def firstTemplate = new AzureVMTemplateBuilder()
-.withName("first-template")
-.withLabels("ubuntu")
-.withLocation("East US")
-.withVirtualMachineSize("Standard_DS2_v2")
-.withNewStorageAccount("<your Storage Account Name>")
-.addNewAdvancedImage()
-    .withReferenceImage("Canonical", "UbuntuServer", "16.04-LTS", "latest")
-    .withInitScript("sudo add-apt-repository ppa:openjdk-r/ppa -y \n" +
-                    "sudo apt-get -y update \n" +
-                    "sudo apt-get install openjdk-8-jre openjdk-8-jre-headless openjdk-8-jdk -y")
-.endAdvancedImage()
-.withAdminCredential("<your admin credential ID>")
-.build()
-
-def myCloud = new AzureVMCloudBuilder()
-.withCloudName("myAzure")
-.withAzureCredentialsId("<your azure credential ID>")
-.withNewResourceGroupName("<your Resource Group Name>")
-.addToTemplates(firstTemplate)
-.addNewTemplate()
-    .withName("second-template")
-    .withLabels("windows")
-    .withLocation("Southeast Asia")
-    .withVirtualMachineSize("Standard_DS2_v2")
-    .withNewStorageAccount("<your Storage Account Name>")
-    .addNewAdvancedImage()
-        .withReferenceImage("MicrosoftWindowsServer", "WindowsServer", "2016-Datacenter", "latest")
-    .endAdvancedImage()
-    .withAdminCredential("<your admin credential ID>")
-.endTemplate()
-.build()
-
-Jenkins.getInstance().clouds.add(myCloud)
-```
-```groovy
-//inherit existing template
-import com.microsoft.azure.vmagent.builders.*
-import com.microsoft.azure.vmagent.*
-
-AzureVMAgentTemplate baseTemplate = new AzureVMTemplateBuilder()
-.withLocation("Southeast Asia")
-.withVirtualMachineSize("Standard_DS2_v2")
-.withStorageAccountType("Premium_LRS")
-.withNewStorageAccount("<your Storage Account Name>")
-    .addNewAdvancedImage()
-         .withReferenceImage("Canonical", "UbuntuServer", "16.04-LTS", "latest")
-    .endAdvancedImage()
-    .withAdminCredential("<your admin credential ID>")
-.build()
-
-def myCloud = new AzureVMCloudBuilder()
-.withCloudName("myAzure")
-.withAzureCredentialsId("<your azure credential ID>")
-.withNewResourceGroupName("<your Resource Group Name>")
-.addNewTemplateLike(baseTemplate)
-    .withName("inherit")
-    .withLabels("inherit")
-    .addNewAdvancedImageLike(baseTemplate.getAdvancedImageInside())
-        .withInitScript("sudo add-apt-repository ppa:openjdk-r/ppa -y \n" +
-                        "sudo apt-get -y update \n" +
-                        "sudo apt-get install openjdk-8-jre openjdk-8-jre-headless openjdk-8-jdk -y")
-    .endAdvancedImage()
-.endTemplate()
-.build()
-
-Jenkins.getInstance().clouds.add(myCloud)
-
-```
-This sample only contains a few arguments of builder, please find all the arguments in folder [builders](src/main/java/com/microsoft/azure/vmagent/builders).
+## Configure VM Template using configuration-as-code plugin
```

Comment:
I assume no need to keep the old groovy docs around?

They're much harder to use than configuration-as-code and I don't think many people would continue to use it when they can just configure what they want and then export the config